.appveyor.yml

```diff
@@ -102,6 +102,7 @@ matrix:
 
 install:
     - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+    - pip install -U virtualenv
```

Comment:
Why not install virtualenv and tox within one line?
I'd prefer you use `--update`, since it's more human readable.

Reply:
It's a workaround and easy to git-blame. Plus tox doesn't need to be upgraded.

Maybe we should better put there a comment about why do we really need to upgrade `virtualenv`?

Reply:
Actual, I prefer long-options too, especially in tests when we run something like `invoke(cli, ["--upgrade-package", "foo"])`, cause I always forget what does `-P` mean. 

These `pip -e -r -U` options are so famous and I type it because of a muscle memory 😅
We could normalize all options here and in the other configs in a separate PR. What do you thnk?

Reply:
Also, we can remove this workaround after tox releases a new version, where virtualenv [has bumped to >=16.0.0](https://tox.readthedocs.io/en/latest/changelog.html#vdraft-2019-10-23).